### PR TITLE
#225 Update to Jakarta

### PR DESCRIPTION
**Pull request by radtriste**
**Many thanks for submitting your Pull Request :heart:!**

https://github.com/serverlessworkflow/sdk-java/issues/225

**What this PR does / why we need it**:
With Jakarta introduced into Spring Boot and Quarkus 3, there is a need for projects to support Jakarta.

**Special notes for reviewers**:
I am not sure how to handle that. Should we have `*-jakarta` modules of the sdk ?
or a different version ?

**Additional information (if needed):**

### CODE REVIEW

**Comment by tsurdilo** [comment]
thanks for pr. will run some tests on it

**Review by ricardozanini — APPROVED** [review]

**Review by cdavernas — APPROVED** [review]
LGTM

**Comment by ricardozanini** [comment]
@manick02 do you mind taking a look?
@radtriste sorry, can you rebase again? 🙏

File: pom.xml [hunk 1]
```diff
@@ -48,14 +48,14 @@
         <version.com.networknt>1.0.86</version.com.networknt>
         <version.org.slf4j>1.7.25</version.org.slf4j>
         <version.com.fasterxml.jackson>2.15.2</version.com.fasterxml.jackson>
-        <version.javax.validation>2.0.1.Final</version.javax.validation>
+        <version.javax.validation>3.0.2</version.javax.validation>
```
Comment (manick02) [inline pom.xml]: nit : should this maven property name changed to reflect jakarta - version.jakarta.validation?

**Comment by manick02** [comment]
Looks good to me, apart from a minor nit

File: pom.xml (hunk 1 above)
Comment (radtriste) [reply pom.xml]: updated

**Comment by radtriste** [comment]
@manick02 @ricardozanini all updated

**Review by manick02 — APPROVED** [review]

**Comment by ricardozanini** [comment]
I'll rebase and include this in a follow up 4.1.0 release supporting Java 11 and Jakarta. @manick02 @fjtirado @cdavernas wdyt?

**Comment by manick02** [comment]
Ok

File: pom.xml [hunk 2]
```diff
@@ -46,6 +46,53 @@
     <module>utils</module>
   </modules>
 
+<<<<<<< issue_225
```
Comment (ricardozanini) [inline pom.xml]:
```suggestion
```

File: pom.xml [hunk 3]
```diff
@@ -46,6 +46,53 @@
...
+      <version.org.mockito>5.12.0</version.org.mockito>
+      <version.org.slf4j>2.0.13</version.org.slf4j>
+      <version.plantuml>8059</version.plantuml>
+      <version.thymeleaf>3.1.2.RELEASE</version.thymeleaf>
+             
+
+      <!-- Checkstyle props -->
+      <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
+      <checkstyle.header.template><![CDATA[
+=======
```
Comment (ricardozanini) [inline pom.xml]:
```suggestion
```

File: pom.xml [hunk 4]
```diff
@@ -91,6 +138,7 @@
     <!-- Checkstyle props -->
     <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
     <checkstyle.header.template><![CDATA[
+>>>>>>> main
```
Comment (ricardozanini) [inline pom.xml]:
```suggestion
```

File: pom.xml [hunk 5]
```diff
@@ -228,6 +276,71 @@
     </dependencies>
   </dependencyManagement>
 
+<<<<<<< issue_225
```
Comment (ricardozanini) [inline pom.xml]:
```suggestion
```

File: pom.xml [hunk 6]
```diff
@@ -228,6 +276,71 @@
...
+                <groupId>net.sourceforge.plantuml</groupId>
+                <artifactId>plantuml</artifactId>
+                <version>${version.plantuml}</version>
+            </dependency>
+            <dependency>
+                <groupId>guru.nidi</groupId>
+                <artifactId>graphviz-java</artifactId>
+                <version>${version.graphviz}</version>
+            </dependency>
+=======
```
Comment (ricardozanini) [inline pom.xml]:
```suggestion
```

File: pom.xml [hunk 7]
```diff
@@ -268,6 +381,7 @@
         </configuration>
       </plugin>
     </plugins>
+>>>>>>> main
```
Comment (ricardozanini) [inline pom.xml]:
```suggestion
```

**Pull request merged**